Cargo.toml

```diff
@@ -165,7 +165,65 @@ repository = "https://github.com/aptos-labs/aptos-core"
 rust-version = "1.64"
 
 [workspace.dependencies]
-# BEGIN MOVE DEPENDENCIES
```

Comment:
This is for tooling (that is why BEGIN and END), and I needed to bring it back.

Reply:
Aah, sorry @wrwg! 😭 I'll send out a PR to fix it now. I assume that whatever I've broken it lives outside of our repo and/or doesn't have tests?

Reply:
https://github.com/aptos-labs/aptos-core/pull/5565